TypeCobol/Compiler/CupParser/NodeBuilder/ProgramClassBuilderNodeListener.cs

```diff
@@ -27,14 +22,15 @@ public virtual void OnLevel1Definition(DataDefinition level1Node)
 
         public virtual void Enter(Node node)
         {
+
         }
 
         public virtual void Exit(Node node)
         {
 
         }
 
-        public virtual void AddNextSentenceStatement([NotNull] NextSentenceStatement stmt)
+        public virtual void AddNextSentenceStatement(NextSentenceStatement stmt)
```

Comment:
Why did you remove [NotNull] ?

Reply:
The `[NotNull]` attribute is set at interface level and is redundant here (ReSharper warning).

Reply:
Ok